source/v4/users.yaml

```diff
@@ -581,3 +581,39 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+
+  /users/{user_id}/verify/email:
+    post:
+      tags:
+        - users
+      summary: Verify User sign-up
+      description: |
+        Verify User account and mark as verified in the database.
```

Comment:
Can we make this `Verify the email used by a user to sign-up their account with.`?